main/include/SettingsSystem.h

```diff
@@ -8,6 +8,8 @@
 
 // Maestro Settings
 #define MAESTRO_SERIAL_BAUD_RATE        9600
+#define MAESTRO_BODY_ID                 1
+#define MAESTRO_DOME_ID                 2
```

Comment:
Do these match what is defined in the Maestro Control Center software for the Body and Dome Maestro boards?

What are acceptable values to put in here?